test/setup-ca-kubernetes.sh

```diff
@@ -64,11 +64,11 @@ EOF
 }
 
 mkdir -p $WORKDIR 2> /dev/null && cd $WORKDIR
-trap cleaup INT
+trap cleanup INT EXIT
 cleanup()
 {
     echo "Cleaning up..."
-    cd .. && rm -rf $WORKDIR 2> /dev/null
```

Comment:
Just above cleanup() we CD into `$WORKDIR`, so it is safe to come out of the directory before removing it.

Saving script originated directory and moving back to that on cleanup and removing `WORKDIR` would be the right solution in my opinion.

Reply:
ah yes, I missed the point that we changed into WORKDIR, sure it is more correct to get out, although it seems to run without error if we do not.
I think pushd/popd could be used here, I will push a new version.